From ab2ca12cd47af87fa82c63b60e472a49a9ce191b Mon Sep 17 00:00:00 2001
From: Duc Dang <dhdang@apm.com>
Date: Wed, 14 Oct 2015 16:56:03 -0700
Subject: [PATCH 09/17] arm64: dts: Add APM X-Gene v1 SoC GFC GPIO controller
 DTS entries

This patch adds the flash controller muxed gpio dts node for
APM X-Gene v1 SoC platforms.

Signed-off-by: Duc Dang <dhdang@apm.com>
---
 arch/arm64/boot/dts/apm/apm-storm.dtsi | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/arch/arm64/boot/dts/apm/apm-storm.dtsi b/arch/arm64/boot/dts/apm/apm-storm.dtsi
index 72680fa..5af9c1c 100644
--- a/arch/arm64/boot/dts/apm/apm-storm.dtsi
+++ b/arch/arm64/boot/dts/apm/apm-storm.dtsi
@@ -725,6 +725,13 @@
 			clocks = <&sdioclk 0>, <&ahbclk 0>;
 		};
 
+		gfcgpio: gfcgpio0@1701c000 {
+			compatible = "apm,xgene-gpio";
+			reg = <0x0 0x1701c000 0x0 0x40>;
+			gpio-controller;
+			#gpio-cells = <2>;
+		};
+
 		phy1: phy@1f21a000 {
 			compatible = "apm,xgene-phy";
 			reg = <0x0 0x1f21a000 0x0 0x100>;
-- 
2.6.2

